AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL13/PhoneCall+Schema.swift

```diff
@@ -30,8 +30,8 @@ extension PhoneCall {
     
     model.fields(
       .field(phoneCall.id, is: .required, ofType: .string),
-      .belongsTo(phoneCall.caller, is: .required, ofType: Person.self, targetNames: ["callerId"]),
-      .belongsTo(phoneCall.callee, is: .required, ofType: Person.self, targetNames: ["calleeId"]),
+      .belongsTo(phoneCall.caller, is: .required, ofType: Person.self, associatedWith: Person.keys.id, targetNames: ["callerId"]),
+      .belongsTo(phoneCall.callee, is: .required, ofType: Person.self, associatedWith: Person.keys.id, targetNames: ["calleeId"]),
```

Comment:
revert this since codegen is not doing to generate this, and in other places as well